### PR TITLE
[Fix/TFlite] Change tflite soname for grep in few unittests

### PR DESCRIPTION
**Pull request by dongju-chae**
This patch changes tflite soname for grep in few unitests.

It resolves https://github.com/nnstreamer/nnstreamer/issues/2944

Signed-off-by: Dongju Chae <dongju.chae@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed []Skipped
2. Run test: [*]Passed [ ]Failed []Skipped

### CODE REVIEW

**Review by jaeyun-jung — APPROVED** [review]

File: tests/nnstreamer_decoder_image_labeling/runTest.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
 PATH_TO_LABEL="../test_models/labels/labels.txt"
 PATH_TO_IMAGE="../test_models/data/orange.png"
 
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=\"${PATH_TO_IMAGE}\" ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw, format=RGB, framerate=0/1 ! tensor_converter ! tensor_filter framework=\"tensorflow-lite\" model=\"${PATH_TO_MODEL}\" ! \
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=\"${PATH_TO_IMAGE}\" ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw, format=RGB, framerate=0/1 ! tensor_converter ! tensor_filter framework=\"tensorflow1-lite\" model=\"${PATH_TO_MODEL}\" ! \
```
Comment (again4you) [inline tests/nnstreamer_decoder_image_labeling/runTest.sh:75]: If you want to test anything either `TF1-lite` or `TF2-lite`, then use just 'tensorflow-lite`
You really want to test `TF1-lite` filter, then it's OK.

File: tests/nnstreamer_decoder_image_segment/runTest.sh [hunk 2]
```diff
@@ -83,7 +83,7 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
 videotestsrc num_buffers=4 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
     t. ! queue ! mix. \
     t. ! queue ! tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
-        tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} latency=1 throughput=1 ! \
+        tensor_filter framework=tensorflow1-lite model=${PATH_TO_MODEL} latency=1 throughput=1 ! \
```
Comment (again4you) [inline tests/nnstreamer_decoder_image_segment/runTest.sh:86]: ditto.

File: tests/nnstreamer_decoder_image_segment/runTest.sh [hunk 3]
```diff
@@ -93,7 +93,7 @@ gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} \
 videotestsrc num_buffers=1 ! videoconvert ! videoscale ! video/x-raw,format=RGB,width=257,height=257 ! tee name=t \
     t. ! queue ! mix. \
     t. ! queue ! tensor_converter ! tensor_transform mode=arithmetic option=typecast:float32,div:255.0 ! \
-        tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} ! \
+        tensor_filter framework=tensorflow1-lite model=${PATH_TO_MODEL} ! \
```
Comment (again4you) [inline tests/nnstreamer_decoder_image_segment/runTest.sh:96]: ditto.

File: tests/nnstreamer_filter_tensorflow_lite/runTest.sh [hunk 4]
```diff
@@ -70,35 +70,35 @@ PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
 PATH_TO_LABEL="../test_models/labels/labels.txt"
 PATH_TO_IMAGE="../test_models/data/orange.png"
 
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} ! filesink location=tensorfilter.out.log" 1 0 0 $PERFORMANCE
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow1-lite model=${PATH_TO_MODEL} ! filesink location=tensorfilter.out.log" 1 0 0 $PERFORMANCE
```
Comment (again4you) [inline tests/nnstreamer_filter_tensorflow_lite/runTest.sh:73]: ditto.

File: tests/nnstreamer_filter_tensorflow_lite/runTest.sh [hunk 5]
```diff
@@ -70,35 +70,35 @@ PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
...
 python checkLabel.py tensorfilter.out.log ${PATH_TO_LABEL} orange
 testResult $? 1 "Golden test comparison" 0 1
 
 # Fail test for invalid input properties
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} input=7:1 inputtype=float32 ! filesink location=tensorfilter.out.log" 2F_n 0 1 $PERFORMANCE
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow1-lite model=${PATH_TO_MODEL} input=7:1 inputtype=float32 ! filesink location=tensorfilter.out.log" 2F_n 0 1 $PERFORMANCE
```
Comment (again4you) [inline tests/nnstreamer_filter_tensorflow_lite/runTest.sh:78]: ditto.

File: tests/nnstreamer_filter_tensorflow_lite/runTest.sh [hunk 6]
```diff
@@ -70,35 +70,35 @@ PATH_TO_MODEL="../test_models/models/mobilenet_v1_1.0_224_quant.tflite"
...
 
 # Fail test for invalid output properties
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow-lite model=${PATH_TO_MODEL} output=1:7 outputtype=int8 ! filesink location=tensorfilter.out.log" 3F_n 0 1 $PERFORMANCE
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} filesrc location=${PATH_TO_IMAGE} ! pngdec ! videoscale ! imagefreeze ! videoconvert ! video/x-raw,format=RGB,framerate=0/1 ! tensor_converter ! tensor_filter framework=tensorflow1-lite model=${PATH_TO_MODEL} output=1:7 outputtype=int8 ! filesink location=tensorfilter.out.log" 3F_n 0 1 $PERFORMANCE
```
Comment (again4you) [inline tests/nnstreamer_filter_tensorflow_lite/runTest.sh:81]: ditto.

**Comment by again4you** [comment]
I will update this rule on our document.

### Basic rule for TF-lite filter

* TF2-Lite with GPU Delegate
`tensor_filter framework=tensorflow2-lite accelerator=true:gpu`

* TF2-Lite without GPU Acceleration
`tensor_filter framework=tensorflow2-lite`

* TF2-Lite or TF-Lite (TF2-Lite is higher priority)
`tensor_filter framework=tensorflow-lite`
`tensor_filter framework=tensorflow-lite accelerator=true:gpu` (If TF-Lite is only supported, then `accelerator` is ignored)

**Review by myungjoo — COMMENTED** [review]
They should be compatible with systems with either tf1-lite and tf2-lite.

File: tests/nnstreamer_decoder_image_labeling/runTest.sh (hunk 1 above)
Comment (dongju-chae) [reply tests/nnstreamer_decoder_image_labeling/runTest.sh:75]: Yes, it's only for tflite1. tflite2 is tested on `tests/nnstreamer_filter_tensorflow2_lite`

**Comment by dongju-chae** [comment]
@again4you @myungjoo I know the changed tflite rule recently. But I think this test should be only for tflite1, not tflite2.

Please refer to `tests/meson.build`
https://github.com/nnstreamer/nnstreamer/blob/386b56722dc7bf39885dc6e8cb644b2455e9d8bf/tests/meson.build#L205-L212

You know tflite1 and tflite2 are installed in CI server. If you use `framework=tensorflow-lite`, we cannot test tflite1.

**Comment by again4you** [comment]
After talking with @dongju-chae and @jdayun-jung, this version is for `TF1-lite`. So, in my opinion, it's OK.
Next time, I will add more test cases including GPU Delegate functionality.

**Review by again4you — APPROVED** [review]

**Pull request merged**